run-all-tests.sh

```diff
@@ -0,0 +1,6 @@
+dirs=$(find collectors/ -type d -maxdepth 1)
+echo $dirs
+for d in $dirs
+do
+    ( cd "$d" && echo "\n**********\n\nrunning tests for $d\n\n************\n\n" && npm run test )
```

Comment:
This doesn't work for me unfortunately, it parses dirs as one big string. I just tried it manually in paws-collector dir
```
 dirs=$(find collectors/ -type d -maxdepth 1)
echo $dirs
for d in $dirs
do
    ( cd "$d" && echo "\n**********\n\nrunning tests for $d\n\n************\n\n" && npm run test )
done

collectors/
collectors//template
collectors//gsuite
collectors//o365
collectors//okta
collectors//auth0
collectors//googlestackdriver
cd: no such file or directory: collectors/\ncollectors//template\ncollectors//gsuite\ncollectors//o365\ncollectors//okta\ncollectors//auth0\ncollectors//googlestackdriver
```

Reply:
hmmm thats odd.

are you in the project root?